vit_vqgan/modeling_vit_vqgan.py

```diff
@@ -470,14 +465,17 @@ def setup(self):
             self.to_patches = FeedForwardLayer(
                 dim1=self.config.intermediate_size,
                 dim2=input_dim,
-                activation="tanh",
+                activation=self.config.hidden_act,
                 config=self.config,
                 dtype=self.dtype,
             )
+        self.encoder_ln = nn.LayerNorm(epsilon=self.config.layer_norm_eps, dtype=self.dtype, use_scale=False)
 
     def encode(self, pixel_values, deterministic: bool = True):
         hidden_states = self.encoder(pixel_values, deterministic=deterministic)
-        hidden_states = self.factor_in(hidden_states, deterministic=deterministic)
+        hidden_states = ACT2FN[self.hidden_act](hidden_states)
```

Comment:
Not sure if we need activation after the encodr. Any particualar reason ?

Reply:
I typically don't like 2 consecutive dense layers as they could be replaced with 1 dense layer (and less total weights) but maybe I'm just picky.
I can remove it if you want.

Reply:
fine with me.